docs/source/txio_board.rst

```diff
@@ -0,0 +1,454 @@
+sources of power, including fan power, optional GPS unit power, and AUX power. As well, the
+daughterboard interface has 6V power routed through it.
+
+Motherboard
+-----------
+From the open source schematics at https://files.ettus.com/schematics/ for the N200, there are
+several images below showing the power routing available on the motherboard:
+
+.. figure:: img/txio/n200_voltages.jpg
+   :scale: 50 %
+   :alt: N200 Unprotected Voltage to Unregulated Voltage
+   :align: center
+
+
+First, the unprotected front panel power input is protected with the above circuitry, and the
+resulting net (supply) is called Vunreg for unregulated voltage supply.
+
+.. figure:: img/txio/n200_6v.jpg
+   :scale: 50 %
+   :alt: N200 Unregulated Voltage to 6V Supply
+   :align: center
+
+Next, this unregulated supply is isolated from other circuitry by way of an L-C circuit shown above.
+It is still unregulated at this point, but the resulting net is called 6V due to the input power to
+the N200 being a 6V wall supply.
+
+.. figure:: img/txio/n200_front_panel_pwr_j102.jpg
+   :scale: 50 %
+   :alt: N200 Front Panel Power Input to ALT Power (Unregulated) On J102
+   :align: center
+
+
+There is also a connector known as “ALT Power” at the back of the motherboard, which is protected
+via fuse F102. This is also the circuitry that creates the Vunprot net, which is simply the power
+input to the N200, protected by a fuse F101, a zener diode package D103 and a capacitor C1170.
+The fuse is 3A, indicated by a silkscreen on the motherboard near connector J102. There is also
+silkscreen that states “AUX Power 6.3V ONLY 500mA”. Therefore, the fuse choice for the TXIO board
+was set to 500mA to protect the TXIO board, and be well within the 3A limit of fuse F102.
+
+LFRX Daughterboard
+------------------
+
+The 6V net is routed through both of the daughterboard interfaces, but first it goes through a
+similar L- C circuit as above for the Vunreg to 6V nets. See the excerpt from the N200 schematics
+below:
+
+.. figure:: img/txio/n200_lfrx_6v.jpg
+   :scale: 50 %
+   :alt: N200 RX Daughterboard Interface 6V Power
+   :align: center
+
+.. _gpio-logic:
+
+GPIO Logic Levels
+-----------------
+The GPIO pins that are routed through the daughterboards on the Ettus N200 are not protected in any
+way, they are a direct connection from the Xilinx FPGA pins, which have LVCMOS signaling. From the
+datasheet, the absolute maximum voltage on any pin is 4.6V:
+
+.. figure:: img/txio/n200_gpio_logic.jpg
+   :scale: 50 %
+   :alt: N200 FPGA Absolute Maximum Ratings. Input Voltage Max 4.6V
+   :align: center
+
+-----------
+TXIO design
+-----------
+This section describes some of the design of the TXIO board, latest revision.
+Please see the pdf of the schematic here: :download:`TXIO schematic <doc/txio_rev5_schematic.pdf>`
+See also a representative BOM here: :download:`TXIO BOM revision 5 <doc/txio_rev5_bom.pdf>`
+
+Change Notes from Revision 4
```

Comment:
I feel like this would fit better near the historical designs section, or after the current revision is described